### ISSUE TITLE
[BUG] pc.headloss_fric can't handle arrays as inputs

### ISSUE THREAD

**Issue by monroews** [AguaClara-Reach/aguaclara#165]
```python
HL_model = pc.headloss_fric(Q_array, D, L, Nu, roughness)
```

I get the following error. I suspect that the wrapper that handles arrays has either been removed or is no longer working.
---------------------------------------------------------------------------
```bash
TypeError                                 Traceback (most recent call last)
<ipython-input-87-ce330e19f995> in <module>()
----> 1 HL_model = pc.headloss_fric(Q_array, D, L, Nu, roughness)

C:\Users\mw24\Anaconda3\lib\site-packages\pint\registry_helpers.py in wrapper(*values, **kw)
    170             new_values, values_by_name = converter(ureg, values, strict)
    171 
--> 172             result = func(*new_values, **kw)
    173 
    174             if container:

c:\users\mw24\github\aguaclara\aguaclara\core\physchem.py in headloss_fric(FlowRate, Diam, Length, Nu, PipeRough)
    230     return (fric(FlowRate, Diam, Nu, PipeRough)
    231             * 8 / (gravity.magnitude * np.pi**2)
--> 232             * (Length * FlowRate**2) / Diam**5
    233             )
    234 

C:\Users\mw24\Anaconda3\lib\site-packages\pint\registry_helpers.py in wrapper(*values, **kw)
    170             new_values, values_by_name = converter(ureg, values, strict)
    171 
--> 172             result = func(*new_values, **kw)
    173 
    174             if container:

c:\users\mw24\github\aguaclara\aguaclara\core\utility.py in wrapper(*args, **kwargs)
    116                         #sizes and shapes of the lists passed.
    117                         result.append(wrapper(*argsList,
--> 118                                               HandlerResult=HandlerResult, **kwargs))
    119                         iterant += 1
    120                 #HandlerResult allows the user to specify what type to

c:\users\mw24\github\aguaclara\aguaclara\core\utility.py in wrapper(*args, **kwargs)
     88             #the function.
     89             if len(sequences) == 0:
---> 90                 result = func(*args, **kwargs)
     91             else:
     92                 #iterant keeps track of how many times we've iterated and

TypeError: fric() got an unexpected keyword argument 'HandlerResult'
```

**Comment by oliver-leung** [comment]
This bug will be resolved in the next release (v0.0.20) this week!

**Comment by monroews** [comment]
Yeah! Good work!
